src/shared/logger.ts

```diff
@@ -202,33 +206,35 @@ function makeDateString(type: 'filename' | 'logfile'): string {
     const isFilename: boolean = type === 'filename'
 
     return `${d.getFullYear()}${isFilename ? '' : '-'}` +
-    // String.prototype.padStart() is not available in Typescript...
-    `${padNumber(d.getMonth() + 1)}${isFilename ? '' : '-'}` +
-    `${padNumber(d.getDate())}${isFilename ? 'T' : ' '}` +
-    `${padNumber(d.getHours())}${isFilename ? '' : ':'}` +
-    `${padNumber(d.getMinutes())}${isFilename ? '' : ':'}` +
-    `${padNumber(d.getSeconds())}`
+        // String.prototype.padStart() is not available in Typescript...
```

Comment:
This isn't related to TypeScript--`padStart` was introduced in ES7, but we target ES6. Can you update the comment accordingly?

Reply:
That's not my code/comment. But sure, updated.